src/qibolab/sweeper.py

```diff
@@ -87,12 +81,13 @@ class Sweeper:
     """
 
     parameter: Parameter
-    values: npt.NDArray
-    pulses: Optional[list] = None
-    channels: Optional[list] = None
-    type: Optional[SweeperType] = SweeperType.ABSOLUTE
+    values: Optional[npt.NDArray] = None
+    linspace: Optional[tuple[float, float, float]] = None
```

Comment:
```suggestion
    range: Optional[tuple[float, float, float]] = None
```
?

@andrea-pasquale, @Edoardo-Pedicillo any advice for a better name than those two?

Reply:
What are `values` and `linspace` in this case?

Reply:
Two different ways to define the sweeper values. `values` is an array containing all the points, same as in 0.1. `range` will be a tuple with three elements `(start, stop, step)` and the values will be defined as `np.arange(start, stop, step)`. Only one of them should be given when initializing the `Sweeper`. If both are given it will raise an error.

Reply:
Changed in 2726f2783916277eeabcf69b85986f90f72b04dc.

I forgot to say above that the idea is that `range` should be used whenever possible, as it is expected to be more efficient for most instruments. It should also be sufficient for most qibocal routines. `values` is still there for special cases where non-equidistant points need to be swept for some reason.

Reply:
Thanks for clarifying, I would say that `range` is fine.